packages/cli/generators/relation/templates/controller-relation-template-belongs-to.ts.ejs

```diff
@@ -0,0 +1,34 @@
+import {
+  repository,
+} from '@loopback/repository';
+import {
+  param,
+  get,
+} from '@loopback/rest';
+import { <%= sourceModelClassName %>, <%= targetModelClassName %> } from '../models';
```

Comment:
We don't use spaces around curly braces.

```suggestion
import {<%= sourceModelClassName %>, <%= targetModelClassName %>} from '../models';
```

Reply:
Done.